eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -1,25 +1,27 @@
 parameters:
-- name: PackageName
-  type: string
-  default: 'not-specified'
 - name: ServiceName
   type: string
   default: 'not-specified'
 - name: ForRelease
-  type: boolean
-  default: false
+  type: string
+  default: ''
+- name: Artifacts
+  type: object
+  default: []
 
 steps:
-  - task: Powershell@2
-    inputs:
-      filePath: $(Build.SourcesDirectory)/eng/common/scripts/Verify-ChangeLog.ps1
-      arguments: >
-        -PackageName ${{ parameters.PackageName }}
-        -ServiceName ${{ parameters.ServiceName }}
-        -RepoRoot $(Build.SourcesDirectory)
-        -RepoName $(Build.Repository.Name)
-        -ForRelease $${{ parameters.ForRelease }}
-      pwsh: true
-      workingDirectory: $(Pipeline.Workspace)
-    displayName: Verify ChangeLogEntry for ${{ parameters.PackageName }}
-    continueOnError: false
\ No newline at end of file
+  - template: /eng/common/pipelines/templates/steps/run-specific-variables.yml
+  - ${{ each artifact in parameters.Artifacts }}:
```

Comment:
Is it easier to do the each loop over the artifacts here or have the caller do it around the template? I sort of feel like we should make the caller do so we aren't indirectly depending on the Artifact structure, as there is likely going to be places where we don't have that (i.e. iOS, C repos).